client/call.go

```diff
@@ -39,3 +40,133 @@ func (c *Call) resetClientCmdID(clock Clock) {
 		Random:   rand.Int63(),
 	}
 }
+
+// Method returns the name of the database command for the call.
+func (c *Call) Method() string {
+	return c.Args.Method()
+}
+
+// GetCall returns a Call object initialized to get the value at key.
+func GetCall(key proto.Key, reply *proto.GetResponse) *Call {
```

Comment:
I add the optional `reply` parameter to all of these call methods to ease the transition of old code. Not sure if I like it, though it does give the caller the opportunity to reduce memory allocations.